app/src/components/Menu/components/StyledIonIcon.ts

```diff
@@ -0,0 +1,23 @@
+import styled from "styled-components";
+import {IonIcon} from "@ionic/react";
```

Comment:
Are we wanting to use Ion Icons or Font Awesome Icons?

Reply:
Not sure, but created the menu item for it because there was already CSS for it

Reply:
Lets stick with Font Awesome Icon, removed file.